website/cue/reference/components/sources/aws_sqs.cue

```diff
@@ -0,0 +1,123 @@
+package metadata
+
+components: sources: aws_sqs: components._aws & {
+	title: "AWS SQS"
+
+	features: {
+		collect: {
+			tls: enabled:        false
+			checkpoint: enabled: false
+			proxy: enabled:      true
+			from: service:       services.aws_sqs
+		}
+		multiline: enabled: false
+		codecs: {
+			enabled:         true
+			default_framing: "bytes"
+		}
+	}
+
+	classes: {
+		commonly_used: true
+		deployment_roles: ["aggregator"]
+		delivery:      "at_least_once"
+		development:   "beta"
+		egress_method: "stream"
+		stateful:      false
+	}
+
+	support: {
+		targets: {
+			"aarch64-unknown-linux-gnu":      true
+			"aarch64-unknown-linux-musl":     true
+			"armv7-unknown-linux-gnueabihf":  true
+			"armv7-unknown-linux-musleabihf": true
+			"x86_64-apple-darwin":            true
+			"x86_64-pc-windows-msv":          true
+			"x86_64-unknown-linux-gnu":       true
+			"x86_64-unknown-linux-musl":      true
+		}
+		requirements: [
+			"""
+					The AWS SQS source requires an SQS queue.
+				""",
+		]
+		warnings: []
+		notices: []
+	}
+
+	installation: {
+		platform_name: null
+	}
+
+	configuration: {
+		acknowledgements: configuration._acknowledgements
+		poll_secs: {
+			common:      true
+			description: "How long to wait when polling SQS for new messages. 0-20 seconds"
+			required:    false
+			warnings: []
+			type: uint: {
+				default: 15
+				unit:    "seconds"
+			}
+		}
+		//  client_concurrency: {
+		//   common: true
+		//   description: "How many clients are receiving / acking SQS messages. Increasing may allow higher throughput. Note: the default is 1 / CPU core"
+		//   required: false
+		//   warnings: []
+		//   type: uint: {
+		//    default: 1
+		//    unit: "# of clients"
+		//   }
+		//  }
```

Comment:
Did we end up leaving this option out?

Reply:
this was not intentional. I'll add this back in.